packages/swap/src/__tests__/integration.test.ts

```diff
@@ -18,20 +18,28 @@ import { QuoteQueryParams } from '@/shared/schemas';
 import {
   ingressEgressEnvironment,
   swappingEnvironment,
+  environment,
 } from '@/shared/tests/fixtures';
 import prisma from '../client';
+import { RpcEnvironment } from '../rpc';
 import app from '../server';
 import { getBrokerQuote } from '../utils/statechain';
 
 jest.mock('../utils/statechain', () => ({ getBrokerQuote: jest.fn() }));
 
 jest.mock('axios', () => ({
   post: jest.fn((url, data) => {
-    if (data.method === 'cf_swapping_environment')
-      return Promise.resolve({ data: swappingEnvironment() });
-
-    if (data.method === 'cf_ingress_egress_environment')
-      return Promise.resolve({ data: ingressEgressEnvironment('0x123') });
+    if (data.method === 'cf_environment') {
+      return Promise.resolve({
+        data: {
+          result: {
+            ...environment().result,
+            swapping: swappingEnvironment().result,
+            ingress_egress: ingressEgressEnvironment('0x123').result,
```

Comment:
no need for these values, we can just use `environment()`

Reply:
yeah i was just trying to make it work fast. environment just clones the structure without injecting anything